### ISSUE TITLE
abdominal aortic dissection

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6250]
**Preferred term label:**
abdominal aortic dissection

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

request by prof. G. Jondeau

**Comment by pnrobinson** [comment]
adding term
